README.md

```diff
@@ -1,3 +1,23 @@
+# janis
+Frontend for [joplin](https://github.com/cityofaustin/joplin).
+
+## Run locally
+
+```
+docker-compose build
```

Comment:
How do you feel about adding a `scripts/serve-local.sh` file that can contain whatever commands need to be run? If we use that consistently across projects it'll be an easy way to start working on a new project.

Reply:
The only question I have about this from looking at the `serve-local.sh` script you have in joplin, it seems like you would have to run docker build each time you start the local server. Is that a problem at all?